src/nft/hooks/useCart.ts

```diff
@@ -0,0 +1,20 @@
+import create from 'zustand'
+import { devtools } from 'zustand/middleware'
+
+type cartState = {
```

Comment:
(here and in general:) types and interfaces should be capitalized.

This should be an interface, not a type. Types should be used for aliases, interfaces should be used for definitions.